docs/book/starter-guide/pipelines/parameters-and-caching.md

```diff
@@ -4,24 +4,25 @@ description: Iteration is native to ZenML.
 
 # Quickly iterating in ZenML
 
-Machine learning pipelines are rerun many times over throughout their development lifecycle. 
+Machine learning pipelines are rerun many times over throughout their development lifecycle.
 
 ## Parameterizing steps
 
-In order to iterate quickly, one must be able to quickly tweak pipeline runs by changing various 
-parameters for the steps that make up your pipeline.
+In order to iterate quickly, one must be able to quickly tweak pipeline runs by
+changing various parameters for the steps that make up your pipeline.
 
-You can configure your pipelines at runtime in the following ways:
-
-- `BaseParameters`: Runtime configuration passed down to steps as parameters.
-- `BaseSettings`: Runtime settings passed down to stack components and pipelines.
-
-In this section, we will focus on `BaseParameters`, and in the Advanced Guide we will 
-[dive deeper into `BaseSettings`](../../advanced-guide/pipelines/pipelines.md).
+{% hint style="info" %}
+If you want to configure runtime settings of pipelines and stack components,
+you'll want to [read the part of the Advanced
+Guide](../../advanced-guide/pipelines/settings.md) where we dive into how to do
+this with `BaseSettings`.
+{% endhint %}
 
-You can parameterize a step by creating a subclass of the `BaseParameters`. When such a 
-config object is passed to a step, it is not treated like other artifacts. Instead, it 
-gets passed into the step when the pipeline is instantiated.
+You can parameterize a step by creating a subclass of the `BaseParameters`. When
+an object like this is passed to a step, it is not treated like other artifacts
+within ZenML. (Artifacts are the inputs and outputs of steps, but these
+parameters objects are handled differently.) Instead, it gets passed into the
```

Comment:
(...) feels a bit weird, perhaps we should just have a short section on artifacts somewhere and then simply link to it?

Reply:
I agree, this doesn't quite explain the difference between artifacts and parameters. I think it needs to be more clear in pointing out that artifacts are information circulated between steps during the pipeline execution, while the parameters are only used in the initial phase, to configure the steps, way before the pipeline is even launched.